src/app/modules/pictureapp-explore/components/edit-userProfile/components/edit-user-photos/edit-user-photos.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Photo } from 'src/app/models/photo';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { MatDialog } from '@angular/material';
+// tslint:disable-next-line:max-line-length
+import { EditUserprofileConfirmationDialogComponent } from '../edit-userprofile-confirmation-dialog/edit-userprofile-confirmation-dialog.component';
+
+@Component({
+  selector: 'app-edit-user-photos',
+  templateUrl: './edit-user-photos.component.html',
+  styleUrls: ['./edit-user-photos.component.scss']
```

Comment:
[2] Why styleUrls value is with brackets, whilst the above one not? Is it because of array?

Reply:
its predefined component element its array and you need to use brackets